libraries/vircadia-client/src/version.h

```diff
@@ -0,0 +1,65 @@
+//
+//  version.h
+//  libraries/client/src
+//
+//  Created by Nshan G. on 18 Feb 2022.
+//  Copyright 2022 Vircadia contributors.
+//
+
+/** \file */
+
+#ifndef VIRCADIA_LIBRARIES_CLIENT_SRC_VERSION_H
+#define VIRCADIA_LIBRARIES_CLIENT_SRC_VERSION_H
+
+#include "common.h"
+
+/**
+ * \brief Defines the layout of version information returned by vircadia_client_version().
+ */
```

Comment:
```suggestion
/// @brief Defines the layout of version information returned by vircadia_client_version().
```
The rest of the source uses the "/// @" style of Doxygen comment and is processed by running the following command in the /tools/doxygen directory:
```
doxygen DoxyFile
```

Reply:
Comment style fixed. The client API documentation can also be built separately with CMake (the Doxyfile is auto-generated), so added instruction for that in the readme.